frontend/ui/widget/progresswidget.lua

```diff
@@ -15,6 +15,8 @@ Configurable attributes:
  * ticks (list)  -- default to nil, use this if you want to insert markers
  * tick_width
  * last  -- maximum tick, used with ticks
+ * margin_v_orig   -- margin_v before adjusting
+ * bordersize_orig -- bordersize before adjusting
```

Comment:
If there are just internal variables, and they shouldn't ever need to be provided by caller, may be don't document them here.
And name them `_orig_margin_v`, with a leading underscore as a kind of convention these are private.